### DISCUSSION TITLE
Pagination on Airtable query

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#8324] [Help] [answered]
Hello, 

Does anyone know if it's possible to have pagination on a table that's pulling in data from an Airtable query? In my current set-up it doesn't look like you can, but would be good to have it confirmed by someone else. 

Thanks!

**Comment by andz-bb** [comment, marked as answer]
hey @adamjdavies, I just tested this on my end and I don't see any way to implement server side pagination with the current Airtable integration. There's support for it in the Airtable API but I don't see any use of that in the current Budibase integration. This looks like a pretty significant limitation considering you can only pull 100 records at a time, so if you like I can create an issue for it. There's currently an issue for a new version of the Airtable integration here: #4122 - maybe pagination could be included as part of this work.

**Reply by ghost** [reply]
Hi @andz-bb, thanks for having a look into this for me. I'll add this as a ticket to the Airtable integration v2.

Thanks!
